### ISSUE TITLE
Is there any need for having formatted text in a note?

### ISSUE THREAD

**Issue by froilo** [xnotepp/xnote#41]
Just to keep track of this idea: having the possibility to format the text in a note might be a nice option, but it seems this would also require quite some work for an unknown benefit.

**Comment by opto** [comment]
yes - add html or markdown edit capability. I have been thinking about this as well.
